apps/core/components/ItineraryCard/itineraryDetail/SectorsList.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+
+import type { SectorsList as SectorType } from './__generated__/SectorsList.graphql';
+import SectorDetail from '../../sectorDetail/SectorDetail';
+
+type Props = {|
+  +data: ?SectorType,
+|};
+
+function SectorsList(props: Props) {
+  const sectors = props.data?.sectors ?? [];
+  return sectors.map((sector, sectorIndex) => {
+    return <SectorDetail key={sectorIndex} data={sector} />;
+  });
+}
+
+export default createFragmentContainer(
+  SectorsList,
+  graphql`
+    fragment SectorsList on Itinerary {
+      sectors {
+        ...SectorDetail
+      }
+    }
+  `,
+);
```

Comment:
also, if you change this fragment to 
```
export default createFragmentContainer(
  SectorsList,
  graphql`
    fragment SectorsList on Sector @relay(plural: true) {
      ...SectorDetail
    }
  `,
);
```

You should be able to reuse the sector list from all of the BookingType components as well

Reply:
Maybe I'm missing something, but how I will then use it for example for `SectorsListReturn`?
Right now `SectorsList` is kind of equal to `SectorsListMulticity` just with different fragment definition.

Reply:
```
unction SectorsListReturn(props: Props) {
  return (
    <>
      <SectorsList data={[props.data?.outbound, props.data?.inbound]} />
    </>
  );
}

export default createFragmentContainer(
  SectorsListReturn,
  graphql`
    fragment SectorsListReturn on BookingInterface {
      ... on BookingReturn {
        inbound {
          ...SectorsList
        }
        outbound {
          ...SectorsList
        }
      }
    }
  `,
);
```

Reply:
Thanks for clarification. Updated and it definitely looks cleaner now.

Reply:
Thanks, looks good now